### PR TITLE
Deployment by using docker in vagrant

### PR DESCRIPTION
**Pull request by satori-do**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+LABEL maintainer="devops@onix-systems.com"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1 \
+    MYSQL_HOSTNAME=localhost \
+    MYSQL_USER=user \
+    MYSQL_PASSWORD=123456 \
+    MYSQL_DATABASE=rename \
+    CONFIG_FILE=/var/www/html/config/config.php
+
```
Comment (canabina) [inline Dockerfile:11]: Why you use the hardcoded variables? I think you should use the one place to store credentials. I seen below "credentials" file, maybe better use it?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,36 @@
+FROM php:7.0-apache
+
+LABEL maintainer="devops@onix-systems.com"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1 \
+    MYSQL_HOSTNAME=localhost \
+    MYSQL_USER=user \
+    MYSQL_PASSWORD=123456 \
+    MYSQL_DATABASE=rename \
+    CONFIG_FILE=/var/www/html/config/config.php
```
Comment (canabina) [inline Dockerfile:10]: You sure that need to use the absolute path name, not a relative?

File: Dockerfile (hunk 1 above)
Comment (satori-do) [reply Dockerfile:11]: These are default values for docker image

File: Dockerfile (hunk 2 above)
Comment (satori-do) [reply Dockerfile:10]: Yes, I am sure, because we always expect config file by this path only inside the docker container

File: Vagrantfile [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+         git \
+         htop \
+         mc \
+         docker-ce
+  curl -L https://github.com/docker/compose/releases/download/1.21.0/docker-compose-$(uname -s)-$(uname -m) -o /usr/local/bin/docker-compose
+  chmod +x /usr/local/bin/docker-compose
+
+  cd /vagrant/
+
+  docker-compose down
```
Comment (mdihtyar) [inline Vagrantfile]: Why do you use `docker-compose down` here? I think, we do not need to use this instruction here, I believe. We have to provide deployment without down time, I think.
Everything will be recreated by `docker-compose up -d` if changes are present

File: Dockerfile [hunk 4]
```diff
@@ -0,0 +1,37 @@
+FROM php:7.0-apache
+
+LABEL maintainer="devops@onix-systems.com"
+
+ENV COMPOSER_ALLOW_SUPERUSER=1 \
+    MYSQL_HOSTNAME=l \
+    MYSQL_USER=u \
+    MYSQL_PASSWORD=1 \
+    MYSQL_DATABASE=r \
```
Comment (mdihtyar) [inline Dockerfile]: Please use more clear default values, smth like `MYSQL_HOSTNAME=localhost`, `MYSQL_USER=user`

File: README.md [hunk 5]
```diff
@@ -3,3 +3,46 @@
...
+Development environment use current folder as source for application.
+
+### Project structure
+```
+    .
+    ├── Dockerfile
+    ├── docker-compose.yml
+    ├── credentials.env          *
+    ├── rmkr.sql            **
+    ├── .env            ***
```
Comment (mdihtyar) [inline README.md]: Please specify, that `.env` file is necessary or no for running the project.

File: README.md [hunk 6]
```diff
@@ -3,3 +3,46 @@
 Сайт [rename.kr.ua](http://rename.kr.ua/) створено, щоб допомогти жителям міста знайти як саме змінилися назви вулиць, згідно с законом про декомунізаціюю
 
 Information about renamed streets in Kirovohrad, Ukraine in 2015-2016
```
Comment (mdihtyar) [inline README.md]: Also I think, readme file should be provided by using only one language, for example `ukrainian`.

File: Dockerfile (hunk 4 above)
Comment (satori-do) [reply Dockerfile]: Ok

File: Vagrantfile (hunk 3 above)
Comment (satori-do) [reply Vagrantfile]: I agree, it is superfluous

File: README.md (hunk 5 above)
Comment (satori-do) [reply README.md]: `***` If these ports are busy - write others in the file `.env` as (.env.example.) ...by default, port 80

Comment (satori-do) [reply README.md]: I will try this way. Excessive explanation can frighten

File: README.md (hunk 6 above)
Comment (satori-do) [reply README.md]: To put it mildly - yes...

File: docker-compose.override.yml [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  mysql:
+    external:
+      name: traefik_webgateway
+  web:
+    external:
+      name: traefik_webgateway
+  labels:
+    - "traefik.enable=true"
+    - "traefik.backend=rename.kr.ua"
+    - "traefik.frontend.rule=Host:renamekr.ua, www.renamekr.ua"
```
Comment (maksym-nazarenko) [inline docker-compose.override.yml]: typo?
renamekr.ua -> rename.kr.ua

Comment (satori-do) [reply docker-compose.override.yml]: Exactly.

File: README.md [hunk 8]
```diff
@@ -1,5 +1,65 @@
...
+http://localhost:8080
+
+---
+### Розгортання проекту на хмарному сервісі, з існуючим зворотнім проксі-сервісом traefik
+
+Для розгортання проекту і підключення до сервіса traefik, необхідно запускати `docker-compose.yml` разом з `docker-compose.override.yml` використовуючи `-f` та змінивши типовий порт в `.env`:
+
+`docker-compose -f docker-compose.yml -f docker-compose.override.yml up -d`
+
+Для тиестування локально, необхідний попередній запуск мережі командою:
```
Comment (mdihtyar) [inline README.md]: `тиестування`

**Review by maksym-nazarenko — APPROVED** [review]
looks good to me

**Comment by mdihtyar** [comment]
Looks good.

**Review by mdihtyar — APPROVED** [review]
Looks good

**Pull request merged**